submission.go

```diff
@@ -55,3 +60,19 @@ func (h *Submission) String() string {
 	comments := fmt.Sprintf("%d comment%s", h.NumComments, plural)
 	return fmt.Sprintf("%d - %s (%s)", h.Score, h.Title, comments)
 }
+
+type Submissions struct {
```

Comment:
Can this be made unexported? It seems like it's only ever used to parse requests and consumers of this library should not need or want access to this type.